eng/scripts/Generate-Patch.ps1

```diff
@@ -120,42 +106,45 @@ function UpdateChangeLog($ArtifactName, $ServiceDirectoryName, $Version) {
   Set-ChangeLogContent -ChangeLogLocation $ChangelogPath -ChangeLogEntries $ChangeLogEntries
 }
 
-if('' -eq $PatchVersion) {
-  $PatchVersion = GetPatchVersion -ReleaseVersion $ReleaseVersion
-  if('' -eq $PatchVersion) {
-    LogError "Could not fetch the patch version. Exitting ..."
-    exit
+function ResetSourcesToReleaseTag($ArtifactName, $ServiceDirectoryName, $ReleaseTag, $RepoRoot, $RemoteName) {
+  $SdkDirPath = Join-Path $RepoRoot "sdk"
+  $ServiceDirPath = Join-Path $SdkDirPath $ServiceDirectoryName
+  $ArtifactDirPath = Join-Path $ServiceDirPath $ArtifactName
+  $ArtifactPomFile = Join-Path $ArtifactDirPath "pom.xml"
+  
+  TestPathThrow -Path $ArtifactDirPath
```

Comment:
Should this be checked before joining with `pom.xml` to create `$ArtifactPomFile`